src/mscorlib/shared/System/Globalization/CompareInfo.cs

```diff
@@ -674,9 +679,14 @@ public virtual bool IsSuffix(string source, string suffix, CompareOptions option
                     SR.ArgumentNull_String);
             }
 
+            return IsSuffix(source.AsReadOnlySpan(), suffix.AsReadOnlySpan(), options);
+        }
+
+        internal virtual bool IsSuffix(ReadOnlySpan<char> source, ReadOnlySpan<char> suffix, CompareOptions options)
```

Comment:
As it's internal, if we don't override it anywhere, it doesn't need to be virtual.